__tests__/integration/helpers/wallet.helper.js

```diff
@@ -381,3 +381,23 @@ export async function waitUntilNextTimestamp(hWallet, txId) {
   loggers.test.log(`Waiting for ${timeToWait}ms for the next timestamp.`);
   await delay(timeToWait);
 }
+
+/**
+ * This method gets the current height of the network
+ * and awaits until the height changes
+ *
+ * We are ignoring the possibility of reorgs here, we
+ * assume that if the height changes, a new block has arrived
+ *
+ * @param {Storage} storage
+ * @returns {Promise<void>}
+ */
+export async function waitNextBlock(storage) {
+  const currentHeight = await storage.getCurrentHeight();
+  let height = currentHeight;
+
+  while (height === currentHeight) {
```

Comment:
Should we implement a timeout here to prevent the CI from running indefinitely? Or does jest timeout by itself?

Maybe we can do something like:

 ```javascript
let elapsedTime = 0;
while (height === currentHeight) {
  if (elapsedTime > TIMEOUT) {
    throw new Error('Timed out waiting for next block.');
  }
  ...
}
```

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/589/commits/2bab139d0273aa21654d1c5010dc4bb8d1c67e41